service/lxd.go

```diff
@@ -20,6 +18,7 @@ import (
 	"golang.org/x/mod/semver"
 
 	"github.com/canonical/microcloud/microcloud/api/types"
+	cloudCli "github.com/canonical/microcloud/microcloud/client"
```

Comment:
Same comment as https://github.com/canonical/microcloud/pull/287/commits/0191050b30712b99814e7e1032c536d27f1b8f05#r1621963740.

Reply:
I've made it `cloudClient`, dropping `micro` to keep it consistent with the other usages of micro* project imports.